### DISCUSSION TITLE
Lab 2 Part 1 Feedback

### DISCUSSION THREAD

**Discussion by shivanidoshi26** [shivanidoshi26/Interactive-Lab-Hub#11] [General]
Include your feedback here!

**Comment by anam884** [comment]
Good, starting point! For the next part maybe you can do the time/date change using one button and background change with the other.  Maybe also try incorporating the analog clock using animations.

**Comment by snlee159** [comment]
Hey Shivani,

Posting the same comments as on Ritika's since you worked together. :) 

I liked the concept of your project and how you switched the different formats. Some thoughts:
* I think it'd be cool if you included some more interactive devices. For example, the Pi has an aux jack so you could hook up a speaker for your idea of a beep
* I like how you switched the formats but I think it'd be cool to take it a step further and "time travel" or something where you could choose the standard format for a country you are "traveling to" and jump to that country's current time and format. 
* Maybe you could add some gamification to it too? Really excited to see what you guys do with this!

**Comment by singhaniasnigdha** [comment]
Hi Shivani!

I like the idea of data/time format changing very much, particularly because it takes so many clicks to do the same on our laptops/phones.

For part 2, I'm interested to see how you will rely on one button for both date and time change and use the other for background. Using the same button for 2 purposes might be tricky, but maybe you could use the rotary encoder or another QWIIC button for the background change. Better still, the background could change automatically when it's night.
